pyvista/core/pointset.py

```diff
@@ -1268,6 +1279,16 @@ def _from_arrays(self, offset, cells, cell_type, points, deep=True):
         points : np.ndarray
             Numpy array containing point locations.
 
+        deep : bool, optional
+            When ``True``, makes a copy of the points array.  Default
+            ``False``.  Cells and cell types are always copied.
+
+        force_float : bool, optional
+            Casts the datatype to float32 if points datatype are
+            non-float.  Default ``True``. Set this to ``False`` to
+            allow non-float types, though this may lead to errors when
+            transforming datasets.
```

Comment:
Same as above.

Reply:
```suggestion
        Casts the datatype to ``float32`` if points datatype are
        non-float.  Default ``True``. Set this to ``False`` to allow
        non-float types, though this may lead to intermediate float
        truncation when rotating datasets.
```